test/visualTest.ts

```diff
@@ -675,6 +688,17 @@ module powerbi.extensibility.visual.test {
                             visualBuilder,
                             ClickEventType.MetaKey,
                             defaultDataViewBuilder.valuesCategory.length,
+                            () => true,
+                            done);
+                    });
+
+                    it("multi selection should work when altKey is pressed", (done) => {
```

Comment:
Becase here we are selecting only first and last items with AltKey and verify amount of all selected items between them